src/nvim/ex_getln.c

```diff
@@ -4641,7 +4638,6 @@ int del_history_idx(int histype, int idx)
     history[histype][i] = history[histype][j];
     i = j;
   }
-  clear_hist_entry(&history[histype][i]);
```

Comment:
We just removed entry `i` by copying entries with index [i+1,idx] (modulo `hislen`) one field to the "left". Why is the entry with index `idx`(`==i`) not cleared anymore, it is now unused?